src/Bridge/Lumen/DocumenterServiceProvider.php

```diff
@@ -0,0 +1,121 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\ApiDocumenter\Bridge\Lumen;
+
+use Illuminate\Contracts\Container\Container;
+use Illuminate\Support\ServiceProvider;
+use LoyaltyCorp\ApiDocumenter\ClassUtils\ClassFinder;
+use LoyaltyCorp\ApiDocumenter\ClassUtils\Interfaces\ClassFinderInterface;
+use LoyaltyCorp\ApiDocumenter\ClassUtils\PropertyInfoExtractor;
+use LoyaltyCorp\ApiDocumenter\Documentation\Generator;
+use LoyaltyCorp\ApiDocumenter\Documentation\Interfaces\GeneratorInterface;
+use LoyaltyCorp\ApiDocumenter\Documentation\Interfaces\RoutesToSchemasConverterInterface;
+use LoyaltyCorp\ApiDocumenter\Documentation\RoutesToSchemasConverter;
+use LoyaltyCorp\ApiDocumenter\Routing\Interfaces\RouteEnhancerInterface;
+use LoyaltyCorp\ApiDocumenter\Routing\Interfaces\RouteExtractorInterface;
+use LoyaltyCorp\ApiDocumenter\Routing\Interfaces\RouteToPathItemConverterInterface;
+use LoyaltyCorp\ApiDocumenter\Routing\ReflectionRouteEnhancer;
+use LoyaltyCorp\ApiDocumenter\Routing\RouteToPathItemConverter;
+use LoyaltyCorp\ApiDocumenter\SchemaBuilders\EntityRequestSchemaBuilder;
+use LoyaltyCorp\ApiDocumenter\SchemaBuilders\Interfaces\OpenApiTypeResolverInterface;
+use LoyaltyCorp\ApiDocumenter\SchemaBuilders\Interfaces\PropertyTypeToSchemaConverterInterface;
+use LoyaltyCorp\ApiDocumenter\SchemaBuilders\Interfaces\SchemaBuilderInterface;
+use LoyaltyCorp\ApiDocumenter\SchemaBuilders\ObjectSchemaBuilder;
+use LoyaltyCorp\ApiDocumenter\SchemaBuilders\OpenApiTypeResolver;
+use LoyaltyCorp\ApiDocumenter\SchemaBuilders\PropertyTypeToSchemaConverter;
+use phpDocumentor\Reflection\DocBlockFactory;
+use Symfony\Component\PropertyInfo\Extractor\PhpDocExtractor;
+use Symfony\Component\PropertyInfo\Extractor\ReflectionExtractor;
+use Symfony\Component\PropertyInfo\PropertyInfoExtractor as BasePropertyInfoExtractor;
+use Symfony\Component\Serializer\NameConverter\CamelCaseToSnakeCaseNameConverter;
+
+/**
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects) Required to configure
+ * @SuppressWarnings(PHPMD.StaticAccess) required to create
```

Comment:
To create? You also don't generally need this as a class level suppression, this can be added directly to the `register()` doc block.

Reply:
Hrm I must have cut off half way through writing this. I did try to put it against the actual call and it didnt get suppressed.

I'll try again